images/Dockerfile.prometheus-exporter

```diff
@@ -0,0 +1,3 @@
+FROM criteord/cassandra_exporter:2.2.1
+
+RUN chown 999:999 /opt/cassandra_exporter/cassandra_exporter.jar
```

Comment:
@viivek46 can you please explain why we need to `chown` the file? What runs as UID 999 and why does it need to own this JAR?

Reply:
> @porridge: can you please explain why we need to chown the file? What runs as UID 999 and why does it need to own this JAR?

Because of the [pod security context](https://github.com/mesosphere/kudo-cassandra-operator/blob/ea8152a05e76fadc525bfe94f85445dee76b6006/operator/templates/stateful-set.yaml#L24-L36) this image container runs under.

Reply:
Thanks @mpereira . However `chown`-ing the file gives the process the permission to modify the code to be run, which is frowned upon from security PoV.
I'm guessing the intent probably to `chmod a+r` it...
````
-rw-rwx--- 1 root root 4607224 Jan 30  2019 cassandra_exporter.jar
```
Right. So the fix should actually be done upstream, let me take a stab at it.

Reply:
https://github.com/criteo/cassandra_exporter/pull/68